modules/EvseManager/IECStateMachine.cpp

```diff
@@ -186,6 +186,9 @@ std::queue<CPEvent> IECStateMachine::state_machine() {
             if (lock_connector_in_state_b) {
                 connector_lock();
             } else {
+                EVLOG_warning <<
+                    "Unlock connector in CP state B. "
```

Comment:
This prints the warning whenever feed_state_machine is called, which could happen any time and more than once.
Maybe the warning in the manifest is enough? I think it is just neccessary to document it with the config option so that people understand that this may cause problems down the road if they don't understand exactly what they are doing

Reply:
Hmm thats correct. Then I chose the wrong place for the warning. But I would still be in favor of a one-time log warning, e.g. when starting the module.

Reply:
A one time message wont hurt in init or so. But we also have other config options to violate the norm in creative ways that only have the warning in the manifest yaml. So I'm ok with either one.

Reply:
I moved the `EVLOG_warning` to the EvseManager ready(). Then this warning is only displayed once at startup.

Reply:
lgtm